packages/quick_actions/example/ios/RunnerUITests/RunnerUITests.m

```diff
@@ -14,19 +14,21 @@ @interface RunnerUITests : XCTestCase
 @implementation RunnerUITests
 
 - (void)setUp {
+  [super setUp];
   self.continueAfterFailure = NO;
 }
 
 - (void)testQuickAction {
   XCUIApplication *app = [[XCUIApplication alloc] init];
   [app launch];
-
+  os_log_error(OS_LOG_DEFAULT, "%@", app.debugDescription);
```

Comment:
Make sure to remove this logging (and below).

Reply:
done